### DISCUSSION TITLE
有点绝望，vmess+https+ws被封

### DISCUSSION THREAD

**Discussion by newflydd** [v2fly/v2ray-core#2124] [General]
用了两年，都相安无事。
开大会之前开始发抽，现在已经完全没法访问。

自己买的服务器，用户也只有我一个人，一个月流量不到2G吧，就平时查查google，看看gmail，玩玩steam什么的。

绝望。

**Comment by Septrum101** [comment]
换个端口？

**Comment by newflydd** [comment]
换了，没用。
我去买订阅了。

thank243 ***@***.***> 于2022年11月4日周五 16:19写道：

**Reply by loophe** [reply]
订阅是什么意思？有那种绑定动态IP的云服务，开机就换一次ip

**Reply by newflydd** [reply]
订阅就是购买第三方专业的梯子服务，购买订阅会动态批量更新第三方分配给你的服务器配置，在v2ryaN客户端界面上方不是有订阅按钮吗
